helm-charts/charts/kube-starrocks/values.yaml

```diff
@@ -1,18 +1,21 @@
 # Default values for kube-starrocks.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+
 global:
   rbac:
+    # if set true, the clusterrole, clusterrolebinding, role, rolebinding, serviceaccount resources will be created for
+    # operator.
     create: true
-  # set the starrockscluster CRD version. from operator v1.6 we have supported v1 CRD and as default CRD version. but the v1alpha1 also supported.
-  # in future v1alpha1 as verify CRD, when the ability is stable will be added into v1.
-  # supported value: v1, v1alpha1
+  # set the starrockscluster CRD version. From operator v1.6 we have supported v1 CRD and as default CRD version, but
+  # the v1alpha1 version is also supported.
+  # supported value: v1, v1alpha1.
   crdVersion: v1
 
 # the starrockscluster crd' name.
 nameOverride: ""
+
 starrocksOperator:
-  # provide a name for operator deployment.
   enabled: true
```

Comment:
can add comments to these fields that are not obvious to end user.

Reply:
the problem is the comment is not correct for `enabled` filed. And It is used together with `global.rbac.create`, I fix the comment there.

Reply:
ok